app/helpers/provider/admin/cms/email_templates_helper.rb

```diff
@@ -6,7 +6,7 @@ def disable_sending_button
 
     link_to "Disable Sending Snippet", '#',
                      onclick: js,
-                     class: 'dangerous-button',
+                     class: 'pf-c-button pf-m-secondary pf-m-danger',
```

Comment:
I made this button "less dangerous" because it just add a snippet to the template, and unless the template is saved, nothing "bad" happens.